features/image-transformations/overlay-using-layers.md

```diff
@@ -0,0 +1,200 @@
+| [q](../image-transformations/resize-crop-and-other-transformations#quality-q)                               | It is used to specify the quality of the output image for lossy formats like JPEG, WebP and AVIF. A large quality number indicates a larger output image size with high quality. A small quality number indicates a smaller output image size with lower quality. It can be a whole number between `1` and `100`. Default value is `80`. |
+
+# Add text over image
+
+You can add any text string over a base image using the following examples. 
+
+You can also control the position of text overlay using these [positional parameters](overlay-using-layers#position-of-layer).
+
+## Usage syntax
+
+### Plain text overlay
+
+Specified with the `i-` parameter in a `text` layer. Supported characters include:
+- All alphabets and numbers (including Unicode letters, marks, and numerals in other languages) 
+- Spaces 
+- Special characters including \_, -, %, !, @, and &
+
+```markup
+https://ik.imagekit.io/demo/base-image.jpg?tr=l-text,i-hello,l-end
+```
+
+### Base64 encoded text overlay
+
+Specified with the `ie-` parameter in a `text` layer. It accepts a base64 string as input, allowing you to overlay special unicode characters (e.g. ©, ®, ™ etc.) that you cannot otherwise pass in a URL as plain string.
+
+{% hint style="info" %}
+1. The base64 string should be made URL safe to ensure that all padding characters(=) are included correctly. In Javascript, a function like `encodeURIComponent()` can be used for this.
+2. Input above the length of 144 characters will be truncated.
+{% endhint %}
+
+```markup
+https://ik.imagekit.io/demo/base-image.jpg?tr=l-text,ie-aGVsbG8%3D,l-end
+```
+Here `aGVsbG8%3D` is the base64 string for "hello".
+
+## Transformation of text overlay
+
+Following transformation parameters are supported on the text inside a layer.
+
+| Parameter                          | Non-layer syntax equivalent                           | Description                           | 
+|------------------------------------|-------------------------------------------------------|---------------------------------------|
+| w                                  | [otw](./overlay#overlay-text-width---otw)             | The maximum width (in pixels) of the overlaid text on the image. The text is wrapped so that any words in a line that go beyond the given width are sent to the next line. Height of the text box is calculated automatically based on the total number of lines. |
+| fs                                 | [ots](./overlay#overlay-text-size---ots)              | It is used to specify the font size.  |
+| ff                                 | [otf](./overlay#overlay-text-font---otf)              | It is used to specify the font of the overlaid text on the image. You can choose [any font from this list](supported-text-font-list#in-built-fonts) or use a [custom font](overlay#using-custom-fonts).                        |
```

Comment:
there should be a custom fonts section on this page. We'll be deprecating the old syntax eventually, and that section uses `otf` based transformation string to illustrate.

Reply:
Added

Reply:
link to 'custom fonts' is still the old one

Reply:
corrected